test/functional/integreatly_test.go

```diff
@@ -41,4 +42,23 @@ func TestIntegreatly(t *testing.T) {
 			})
 		}
 	})
+
+	// Do not execute these tests unless NIGHTLY is set to true
+	if os.Getenv("NIGHTLY") != "true" {
+
+		t.Logf("Skipping Nightly tests as NIGHTLY env var is not set to true")
```

Comment:
Use `t.Skip` to report that that the suite has been skipped

Reply:
+1, I wanted to suggest the same, docs ;) - https://golang.org/pkg/testing/#T.Skip